Eatery Blue/Eatery Blue/UI/Profile/ProfileViewController.swift

```diff
@@ -76,12 +76,16 @@ class ProfileViewController: UIViewController {
         } else {
             viewControllers[0] = viewController
         }
-        theNavigationController.setViewControllers(viewControllers, animated: animated)
```

Comment:
This variable naming is a bit weird. Slight nit but I think consistent naming is important. Either it is a specific navigation control or you can just call it `navigationController`. 
You can learn more about the style guidelines (under naming).

https://www.swift.org/documentation/api-design-guidelines/

Reply:
+1